__tests__/books.spec.js

```diff
@@ -102,18 +102,61 @@ describe('GET ALL BOOKS API ENDPOINT', () => {
         server.close();
     });
 
+    it('should not get book if page query is not a number', async () => {
+        const { status, body } = await server().get('/api/v1/books?page=e');
+
+        expect(status).toBe(422);
+        expect(body).toMatchSnapshot();
+    });
+
+    it('should not get book if limit query is not a number', async () => {
+        const { status, body } = await server().get('/api/v1/books?limit=e');
+
+        expect(status).toBe(422);
+        expect(body).toMatchSnapshot();
+    });
+
     it('should return all books', async () => {
         const firstBook = getBook();
         firstBook.isbn = '128b4v389028074';
         const secondBook = getBook();
         secondBook.isbn = '9204798753002380';
+        const thirdBook = getBook();
```

Comment:
Please use a loop to generate this. A better way is to create 2 items, and pass `limit` to be 1 so we get two pages

Reply:
feedback implemented